components/Vote/Collapsible.js

```diff
@@ -23,6 +28,31 @@ const styles = {
     width: 26,
     marginLeft: -6,
     display: 'inline-block'
+  }),
+  text: css({
```

Comment:
```suggestion
  text: css(Interaction.fontRule, {
```
This will do the child selector for bold properly for you.